File: codigos_de_prueba/prueba0.cpp
#include <iostream>
#include <vector>

using namespace std;

double calcularPromedio(const vector<int>& numeros) {
    if (numeros.empty()) return 0.0;
    
    double suma = 0.0;
    for (int num : numeros) {
        suma += num;
    }
    return suma / numeros.size();
}

int encontrarMaximo(const vector<int>& numeros) {
    if (numeros.empty()) return -1;
    
    int maximo = numeros[0];
    for (size_t i = 1; i < numeros.size(); ++i) {
        if (numeros[i] > maximo) {
            maximo = numeros[i];
        }
    }
    return maximo;
}

int main() {
    vector<int> datos = {12, 45, 23, 67, 8, 34};
    
    cout << "Promedio: " << calcularPromedio(datos) << endl;
    cout << "Valor máximo: " << encontrarMaximo(datos) << endl;
    
    return 0;
}

File: codigos_de_prueba/prueba0.py
def calcular_promedio(numeros):
    total = 0
    cantidad = 0
    for num in numeros:
        total += num
        cantidad += 1
    if cantidad == 0:
        return 0
    return total / cantidad

def encontrar_maximo(numeros):
    maximo = numeros[0]
    for num in numeros:
        if num > maximo:
            maximo = num
    return maximo

lista = [5, 8, 2, 10, 3]
print(f"Promedio: {calcular_promedio(lista)}")
print(f"Máximo: {encontrar_maximo(lista)}")